core/src/main/java/org/elasticsearch/index/query/QueryStringQueryBuilder.java

```diff
@@ -989,8 +1012,11 @@ protected Query doToQuery(QueryShardContext context) throws IOException {
         }
 
         queryParser.setDefaultOperator(defaultOperator.toQueryParserOperator());
+        queryParser.setType(type);
         if (tieBreaker != null) {
             queryParser.setGroupTieBreaker(tieBreaker);
+        } else {
+            queryParser.setGroupTieBreaker(type.tieBreaker());
```

Comment:
minor: Should we set this in `setType()` in the `queryParser` since the value is tied to the type anyway?